src/OfficeHours.ts

```diff
@@ -151,28 +150,28 @@ export default class OfficeHours {
         const canSendMessages = newChannel.permissionsFor(randomUser).has("SEND_MESSAGES");
         if (canSendMessages && !this.data.isOpen)
             this.open(newChannel);
-        else if (canSendMessages && this.data.isOpen) 
+        else if (canSendMessages && this.data.isOpen)
             this.close(newChannel);
     }
 
     sendOnThisDayMessage(channel: Discord.TextChannel) {
-        
+
         const onThisDayMsg = fetch('https://history.muffinlabs.com/date')
-        .then(r => r.json())
-        .then(({ data: { Events } }) => Events[Math.floor(Math.random() * Events.length)])
-        .then(event => { 
-            channel.send(`On this day in ${event.year}, ${event.text}`);
-        })
-        .catch(r => console.warn('Failed to fetch message of the day: ' + r));
+            .then(r => r.json())
+            .then(({ data: { Events } }) => Events[Math.floor(Math.random() * Events.length)])
+            .then(event => {
+                channel.send(`On this day in ${event.year}, ${event.text}`);
+            })
+            .catch(r => console.warn('Failed to fetch message of the day: ' + r));
```

Comment:
Might be cleaner to convert this to an `async` method now that we're here anyway 😉